src/xcode/ENA/ENA/Source/Scenes/AppClosureNoticeDetails/__tests__/AppClosureNoticeDetailsViewModelTests.swift

```diff
@@ -0,0 +1,52 @@
+//
```

Comment:
This test fails on my machine. Is it only me?

Reply:
It is passing on the pipeline.